### PR TITLE
Example pipeline flowsheets

### PR DESCRIPTION
**Pull request by Robbybp**
## Summary/Motivation:
Example flowsheets for the pipeline modeling objects introduced by #667. These are in their own PR to hopefully make things a little easier to review.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by andrewlee94** [comment]
@Robbybp This PR is probably going to been to be updated now that #766 has been merged. All of he gas distribution models have been moved to `idaes/models_extra/gas_distribution` so I expect the imports in this PR are now broken.

**Comment by Robbybp** [comment]
@andrewlee94 Thanks for the notification, I'll update this branch.

**Comment by michaelbynum** [comment]
Will there be conflicts between this PR and #780? Does it matter which one is merged first? I think it probably makes sense to merge #780, then update this PR and make sure tests still pass?

**Comment by Robbybp** [comment]
@michaelbynum I agree. Merge #780, then update and merge (hopefully) this PR. I have another branch that combines this one with #780, and will merge it here once #780 gets merged.

**Comment by ksbeattie** [comment]
@Robbybp any update on this?  Still possible for the May release?

**Comment by Robbybp** [comment]
@ksbeattie yeah definitely possible, just waiting on review. Not necessary though

File: idaes/models_extra/gas_distribution/flowsheets/simple_network_model.py
Comment (blnicho) [inline idaes/models_extra/gas_distribution/flowsheets/simple_network_model.py:238]: Can this TODO comment be removed?

**Review by blnicho — DISMISSED** [review]
I found a couple typos but otherwise this looks fine

**Comment by Robbybp** [comment]
To address why these "examples" are in idaes-pse:
- One of the files just constructs a simple flowsheet using the pipeline unit models and property package. This lives in the `flowsheets` directory, but I often refer to it as an "example" as, to me, the flowsheets we include in the idaes-pse repository are more-or-less examples. 
- The files in the `idaes/apps/nmpc/examples` directory are examples of some utilities I put together for setting up dynamic optimization and NMPC problems. The code is not quite mature/stable enough to warrant an example in examples-pse, but I wanted to commit it, as well as the examples here, for the purpose of pointing other team members to it. I think this is fine as long as it in apps.

**Review by blnicho — APPROVED** [review]

**Comment by ksbeattie** [comment]
@michaelbynum, any estimate on when you can look at this?

**Comment by michaelbynum** [comment]
Unfortunately, it is going to be at least mid-July before I have a chance...

**Comment by Robbybp** [comment]
I'm just going to close this PR, as I don't think having these examples in the repo is necessary (anymore).

**Pull request closed without merging**
